src/contracts/GPv2AllowListAuthentication.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: LGPL-3.0-or-newer
+pragma solidity ^0.7.5;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/EnumerableSet.sol";
+import "./interfaces/GPv2Authentication.sol";
+
+/// @title Gnosis Protocol v2 Access Control Contract
+/// @author Gnosis Developers
+contract GPv2AllowListAuthentication is Ownable, GPv2Authentication {
+    using EnumerableSet for EnumerableSet.AddressSet;
+
+    EnumerableSet.AddressSet private solvers;
+
+    function addSolver(address solver) public onlyOwner {
```

Comment:
These should all be `external` unless they are intended to be called internally as well.

Reply:
It looks like the PR was merged without these changes.